.github/workflows/deploy_doc.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
   deploy-docs:
     needs: [evaluate-label]
-    uses: qibogang/workflows/.github/workflows/deploy-ghpages-latest-stable.yml@tar
+    uses: qibogang/workflows/.github/workflows/deploy-ghpages-latest-stable.yml@tar #TODO: change the branch
```

Comment:
Soon we should start using these hacks to test new workflows, and find a better way.